docs/navigation-advanced.md

```diff
@@ -0,0 +1,334 @@
+# Advanced navigation
+
+This document shows you how to configure the following Luigi features:
+
+* [View groups](#view-groups)
+* [Create a dynamically changeable path](#create-a-dynamic-path)
+* [Contexts](#contexts)
+* [Profile](#profile) 
+* [Context switcher](#contextswitcher) 
+* [Product switcher](d#productswitcher) 
+* [App switcher](#appswitcher) 
+* [Additional options](#additional-options)
+
+
+## View groups
+
+View groups are a way to reuse micro frontends (i.e. views) in Luigi.
+
+Imagine your application hosts two micro frontend views: `http://example.com/a#e` and  `http://example.com/b#f`. Due to hash routing and a different path up to `#`, they are, by default, rendered in different iframes. However, as they both have the **same origin**, such as `example.com`, you want to render them in the same iframe. To achieve that:
+
+1. Define the **viewGroup** parameter for any navigation node.
+2. Children of that node will automatically be considered part of the same view group.
```

Comment:
I think these two do not go well with the introductory line - the first one starting with `define` is fine :) but the second one is not a step. I think maybe it's better to have it in one sentence?

Reply:
Agreed! 🙂